2-mandatory/2-water-bottle.js

```diff
@@ -19,21 +19,73 @@ let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    if (this.volume <= 100) {
```

Comment:
Hello Marina
You need to change <= to <
because if volume was 100 **this.volume <= 100 would be true**
and amount would be changed to 200 which would be incorrect.

Reply:
Yup, i think Delroy made a good point on deciding when to fill up the bottle. Anyway, is the **fillUp** function will fill the bottle up to 100 i.e. **volume = 100**